HMS-9084: update the terminology section in overview (#1217)
content-services/content-sources-backend: docs/overview.md

#### Platform

Even bigger whole that groups multiple applications (Console Dot Platform).

---

### Extended Terminology

#### Advisory

A formal notification from a vendor (e.g., Red Hat) regarding an issue in a software package that requires an update. Advisories provide structured information about the necessary update and are typically categorized by the type of problem they fix (e.g., security, bug fix, enhancement).

#### EPEL & Community Repositories

> **EPEL** (Extra Packages for Enterprise Linux) repositories are read-only repositories sourced from the Fedora Project (e.g., `https://dl.fedoraproject.org/pub/epel/...`). Historically referred to as "Popular Repositories," they are now commonly designated as **Community Repositories** to reflect their automatic, shared availability across users and organizations. They are sometimes referred to as "unified EPEL" or "shared EPEL."

Reviewer: ```suggestion
**EPEL** (Extra Packages for Enterprise Linux) repositories are read-only repositories sourced from the Fedora Project (e.g., `https://dl.fedoraproject.org/pub/epel/...`). In the context of our app, EPEL repositories are available as **Community Repositories** to reflect their automatic, shared availability across users and organizations. 
```
One small comment. EPEL is usually just referred to as EPEL. These other names like "Popular Repositories" and "Community Repositories" are only used in the context of our app. And since unified EPEL is really just the name of the epic, I wouldn't document it.

> #### Errata

Reviewer 2: that makes sense, i think there is a distinction. just found this [article](https://access.redhat.com/articles/explaining_redhat_errata) that states the difference as advisory being the published text and errata being the packaged release. but it also mentions that they're interchangeable :laughing:

#### Platform

Even bigger whole that groups multiple applications (Console Dot Platform).

---

### Extended Terminology

#### EPEL & Community Repositories

> **EPEL** (Extra Packages for Enterprise Linux) is a popular repository maintained by the Fedora Project community (https://dl.fedoraproject.org/pub/epel/...). While EPEL is not an official part of the RHEL subscription, it acts as a crucial conduit, bringing a wide range of packages from the Fedora upstream to the RHEL downstream.

Author: I see. Thanks for clarifying that!
